awx/ui_next/src/screens/Project/shared/ProjectSubForms/SharedFields.jsx

```diff
@@ -84,6 +84,19 @@ export const ScmTypeOptions = withI18n()(
                   of time required to complete an update.`
             )}
           />
+          <CheckboxField
+            id="option-scm-track-submodules"
+            name="scm_track_submodules"
+            label={i18n._(t`Track submodules`)}
+            tooltip={i18n._(
+              t`If yes, submodules will track the latest commit on
```

Comment:
I would change this string just to remove the `If yes,` to be in keeping with other checkbox tooltips.

Reply:
Modified as requested